### ISSUE TITLE
BUG: Flexslider loading fine but images are not displaying

### ISSUE THREAD

**Issue by mhenden** [dynamic/silverstripe-flexslider#216]
**Describe the bug**
Just loaded up Flexslider on a Silverstripe site in a development environment. I've set up two slides to test. A default space has opened up on my page for the slider but the images are not displaying. When I look at the parsed code I can see paths to the correct images, but everything is greyed out (hence the blank space). 

Think I might have missed something? 

Scripts loaded on page are:

<% require javascript('//code.jquery.com/jquery-3.3.1.min.js') %>
<% require themedJavascript('clicky-menus') %> (navigation script)

Do I need anything else? 

**To Reproduce**

**Expected behavior**
Expected to see images

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Mac OSX 10.14.6
 - Browser [e.g. chrome, safari] Firefox
 - Version [e.g. 22] 102.8.0esr

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mhenden** [comment]
Now it's getting seriously weird -- can't see **ANY** scripts loading on this site where I can see scripts in the footer of other sites I've built ?? But site is still functioning as expected (NoCaptcha, etc??)

First time I've set up a slider and -- no images ??!

**Comment by mhenden** [comment]
Have just verified in console that `jquery.flexslider-min.js?` **IS** loading. So it shouldn't be that... ??

**Comment by muskie9** [comment]
Hi @mhenden,

Thanks for the info. To be honest this module, while being a staple in our website builds, has not gotten as much attention as we would have liked. One thing to note from a script perspective is you can disable the module requiring various libraries based on the model the relation extension is applied to [code here](https://github.com/dynamic/silverstripe-flexslider/blob/4.2/src/ORM/FlexSlider.php#L200-L217).

Related to the above link we've "disabled" the module's jquery requirement in favor of a project's jquery requirement through something similar as:

```yml
SilverStripe\CMS\Model\SiteTree:
  jquery_enabled: false
  extensions: 
    - Dynamic\FlexSlider\ORM\ FlexSlider
```

I don't know that this will ultimately resolve your issue, but it does remove the CDN requirement from your project defined by the module.

As a heads up, Flexslider hasn't been updated for some time as a plugin in some time so we are actively working on an alternative "slider" module that will have similar features. A migration plan is in the works as well as we have multiple sites utilizing this module independently, or with the elemental module requiring this as a base module.

**Comment by mhenden** [comment]
Hi @muskie9 

Have tried applying your fix in various ways, without success. In the first instance I put `jquery_enabled:` directly below the namespaced path that I'm using (`App\MyProject\HomePage:`). This resulted in an immediate 'white screen of death'. 

I then placed your code in place of my Flexslider config code and this resulted, unsurprisingly, in an error, due to incorrect namespacing. 

I then placed the first two lines of your code beneath my config code, which left me with the initial error (slider loads but images don't). 

However looking at the link to the source code you sent me, I see that Flexslider is wanting `jquery-3.6.1min.js`, while I have the following jquery call at the bottom of my page:

`<% require javascript('//code.jquery.com/jquery-3.3.1.min.js') %>`

Could this be the root of the problem? 

Would your 'alternative slider module' be the solution, and if so, how far away is it?
